proj/hps_accel/gateware/gen2/accelerator.py

```diff
@@ -57,7 +57,7 @@
     #  The maximum output value
     ('output_activation_max', signed(8)),
     # Address of start of input data, in 16 byte blocks (i.e addr 1 = byte 16)
```

Comment:
I think this comment may also need changing? (now bytes instead of 16 byte blocks)

Reply:
Done. Thank you.